### ISSUE TITLE
The accuracy of C3D training from scartch is low

### ISSUE THREAD

**Issue by Fancy-sf** [jfzhang95/pytorch-video-recognition#32]
The accuracy of C3D training from scratch is 30% with lr=1e-5 and the accuracy is below 1% with lr=1e-3 , which are lower than the paper claimed. I also notice someone added BN to C3D and the accuracy is about 45%. Anybody knows why?

**Comment by civat** [comment]
The acc reported in this repo may be not based on the implementation in this repo. The performance can simply improved by standardizing the input:
1. Normalize input into [0, 1] by x/255;
2. Standardize input using mean [0.485, 0.456, 0.406] and variance [0.229, 0.224, 0.225].

**Comment by civat** [comment]
The acc reported in this repo may be not based on the implementation in this repo. The performance can simply improved by standardizing the input:

Normalize input into [0, 1] by x/255;
Standardize input using mean [0.485, 0.456, 0.406] and variance [0.229, 0.224, 0.225].

After standardizing, the acc is about 50%.

**Comment by skyqwe123** [comment]
@civat What about R2Plus1D? The accuracy of R2Plus1Dtraining from scratch is also very lower (1%)
